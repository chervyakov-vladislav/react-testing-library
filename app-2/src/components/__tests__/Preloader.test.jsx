import { screen, render } from '@testing-library/react';

import { Preloader } from '../Preloader';

describe('Preloader', () => {
  it(('рендер прелоадера'), () => {
    render(<Preloader />);

    expect(screen.getByRole('progressbar')).toBeInTheDocument();
    expect(screen.getByRole('progressbar')).toHaveClass('progress');
    expect(screen.getByRole('progressbar')).toMatchSnapshot();
  });
});