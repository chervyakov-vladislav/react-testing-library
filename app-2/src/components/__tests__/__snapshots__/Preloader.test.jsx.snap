// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Preloader рендер прелоадера 1`] = `
<div
  class="progress"
  role="progressbar"
>
  <div
    class="indeterminate"
  />
</div>
`;
